locales/zh_CN/LC_MESSAGES/dev_install.po

```diff
@@ -7,302 +7,348 @@ msgid ""
 msgstr ""
 "Project-Id-Version: taichi 0.5.14\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-08-02 22:06+0800\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"POT-Creation-Date: 2020-08-07 00:14+0800\n"
+"PO-Revision-Date: 2020-08-09 09:13+0800\n"
 "Language: zh_CN\n"
 "Language-Team: \n"
-"Plural-Forms: nplurals=1; plural=0\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 2.3.1\n"
 
 #: ../../dev_install.rst:4
 msgid "Developer installation"
 msgstr "开发者安装指南"
 
 #: ../../dev_install.rst:6
 msgid ""
-"Note this is for the compiler developers of the Taichi programming "
-"language. End users should use the pip packages instead of building from "
-"source. To build with NVIDIA GPU support, CUDA 10.0+ is needed. This "
-"installation guide works for Ubuntu 16.04+ and OS X 10.14+. For precise "
-"build instructions on Windows, please check out `appveyor.yml "
-"<https://github.com/taichi-dev/taichi/blob/master/appveyor.yml>`_, which "
-"does basically the same thing as the following instructions. We use "
-"MSBUILD.exe to build the generated project. Please note that Windows "
-"could have multiple instances of MSBUILD.exe shipped with different "
-"products. Please make sure you add the path for MSBUILD.exe within your "
-"MSVS directory and make it a higher priority (for instance than the one "
-"shipped with .NET)."
-msgstr ""
+"Note this is for the compiler developers of the Taichi programming language. End "
+"users should use the pip packages instead of building from source. To build with "
+"NVIDIA GPU support, CUDA 10.0+ is needed. This installation guide works for "
+"Ubuntu 16.04+ and OS X 10.14+. For precise build instructions on Windows, please "
+"check out `appveyor.yml <https://github.com/taichi-dev/taichi/blob/master/"
+"appveyor.yml>`_, which does basically the same thing as the following "
+"instructions. We use MSBUILD.exe to build the generated project. Please note "
+"that Windows could have multiple instances of MSBUILD.exe shipped with different "
+"products. Please make sure you add the path for MSBUILD.exe within your MSVS "
+"directory and make it a higher priority (for instance than the one shipped with ."
+"NET)."
+msgstr ""
+"请注意，本章节是为 Taichi 语言编译器的开发人员所准备的。终端用户应该使用 pip 进行"
+"安装，而不是从源代码开始构建。为支持在 NVIDIA GPU 上进行工程构建，CUDA 10.0+是必"
```

Comment:
```suggestion
"安装，而不是从源代码开始构建。要在构建加入对 NVIDIA GPU 的支持，CUDA 10.0+是必"
```
Btw, @yuanming-hu I thought building with CUDA support doesn't need CUDA thanks to our `DynamicLoader`?